src/components/search.js

```diff
@@ -23,7 +23,9 @@ export const Search = ({ selectedMojos, removeFunction, sendFunction }) => (
           />
         </Style.SearchContainer>
         <TouchableOpacity
-          onPress={() => sendFunction(selectedMojos[0].slackID)}
+          onPress={() =>
+            sendFunction(selectedMojos[0].slackID, selectedMojos[0].name)
```

Comment:
We support an array of slack hosts. Could we map this and handle in sendNotification?

Also, I'd prefer `onSend` or `onSendClick` since that implies the caller must pass a callback function.